### DISCUSSION TITLE
Reset axis and origin for the Create 3 Navigate To Position action

### DISCUSSION THREAD

**Discussion by asavic01** [iRobotEducation/create3_docs#379] [Questions]
I want to make my Create 3 go to several unique positions using the Nav2 action, however, I can't figure out how to reset the axis that the robot believes is x and y. My robot needs to be able to go to several locations consistently and I thought the best way to do this would be to call the Nav2 action and pass in those coordinates that correspond to the location, however the robot does not have a consistent x and y axis it follows when I turn it on and off. In addition, I want to zero out where it thinks the origin is (0, 0) and can't find anything on this. Any help would be very much appreciated.

**Comment by alsora** [comment]
First of all, a small correction: the `NavigateToPosition` action that is exposed by the Create 3 robot is not related with the `Nav2` (which is a separate software).

The robot does not have a persistent idea of its location in the space and, if you restart the application (or the robot), the coordinate system will reset to [0, 0, 0].
The `NavigateToPosition` action takes a goal expressed in a global coordinate frame.

We provide a ROS 2 service to reset the pose estimate.
I just realized that it's not documented in the docs, but you can find information here https://github.com/iRobotEducation/create3_docs/discussions/363
